### PR TITLE
Call exitScriBt if ScriBt was exited through Ctrl+C

### PR DESCRIPTION
**Pull request by timschumi**
Also, this includes a fix because I forgot to add slashes at the end of $CALL_ME_ROOT.
Maybe the handling of variables containing paths should be unified (slash at the end or no slash?)

### CODE REVIEW

File: ROM.sh [hunk 1]
```diff
@@ -42,6 +42,12 @@ function cherrypick() # Automated Use only
     echo -e "${CL_WYT}==================================================================${NONE}";
 } # cherrypick
 
+function interrupt()
+{
+    echo -en "\n*** Ouch! Plz don't kill me! ***\n"
+    exitScriBt 1
+}
```
Comment (a7r3) [inline ROM.sh]: A Request, add a comment specifying the name of the function, Just after the }

```
function interrupt()  
{  
#
# the content;  
#
} # interrupt
```  

Part of my coding style, included in all functions :D

**Comment by a7r3** [comment]
Unification would be done if needed, for now, it's set just before it is used.

Important Vars are set on `the_start()` itself

And, Yeah, a forward-slash is needed after, `${CALL_ME_ROOT}`, just saw the error

File: ROM.sh [hunk 2]
```diff
@@ -42,6 +42,12 @@ function cherrypick() # Automated Use only
     echo -e "${CL_WYT}==================================================================${NONE}";
 } # cherrypick
 
+function interrupt()
+{
+    echo -en "\n*** Ouch! Plz don't kill me! ***\n"
```
Comment (a7r3) [inline ROM.sh]: 1. Semicolon at the end
2. Remove `-n` from `echo`'s parameters, so that `\n` at the start of statement is made use of

**Comment by a7r3** [comment]
Good to go! Would review one more time and Merge :+1:

**Comment by a7r3** [comment]
@TimSchumi how would you like to get your name added ?  
It'd be added in ROM.sh and ScriBt wiki  
[Refer here for other Contributors](https://github.com/a7r3/ScriBt/blob/master/ROM.sh#L23)

**Comment by timschumi** [comment]
You decide
But if you add me, could you please add me as
Tim Schumacher (TimSchumi)

**Comment by a7r3** [comment]
@TimSchumi Tim Schumacher is your name right ? (a Confirmation :grin: )

**Comment by timschumi** [comment]
Yes, it's correct.
I would prefer the writing "Tim Schumacher (TimSchumi)" instead of "Tim "TimSchumi" Schumacher". That would sound kinda stupid with that name combination

**Comment by a7r3** [comment]
That's why I asked :laughing:  

Thanks for your Contributions, Keep it up!

**Pull request merged**